packages/frontend/src/components/Game/GameBoard/Tile/index.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './style.scss';
 
-const Tile = ({ photo, onFlip, index, faceUp, id }) => {
+const Tile = ({ photo, onFlip, faceUp, id }) => {
```

Comment:
We had talked about switching these props and the argument that gets passed to the flipTile action back to the entire tile. I think this would make the code a lot more flexible down the road if we needed to add any additional properties to the tile objects. Does this sound reasonable? If so, could we do one more pass at this to pass the entire tile object instead of just isolated values?

Reply:
```javascript const Tile = ({ tile, onFlip }) => {

  // ...
  
  const onClickTile = () => {
    onFlip(tile);
  };

  // ...

}```